test/middleware/retry_test.rb

```diff
@@ -95,30 +112,30 @@ def sleep_amount(retries)
 
     def test_exponential_backoff
       middleware = Faraday::Request::Retry.new(nil, :max => 5, :interval => 0.1, :backoff_factor => 2)
-      assert_equal middleware.sleep_amount(5), 0.1
-      assert_equal middleware.sleep_amount(4), 0.2
-      assert_equal middleware.sleep_amount(3), 0.4
+      assert_equal middleware.sleep_amount(5, {}), 0.1
```

Comment:
We can use the new `calculate_retry_interval` for this and the following tests so we don't need to pass `{}` every time.